app/authProvider.js

```diff
@@ -20,7 +20,7 @@ export class AuthProvider {
         const params = {
           client_id: oidcClientId,
           redirect_uri: `${appBaseUri}/oauth/callback`,
-          scope: 'openid profile',
+          scope: 'openid profile Organisation',
```

Comment:
Do we need this `Organisation` scope? can you try without and see if everything still runs fine without it.

Reply:
`Request failed with status code 401`

Reply:
ok let's leave it in then 😂 must need it for the api calls